src/sandbox/wiktor/LineChart.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useRef } from 'react';
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+} from 'recharts';
+
+interface ILineChartProps {
+  data: { datetime: string; value: number }[];
+}
+
+const LineChartComponent: React.FC<ILineChartProps> = ({ data }) => {
+  const chartRef = useRef<HTMLDivElement>(null);
+
+  const minValue = Math.min(...data.map((d) => d.value));
```

Comment:
https://lodash.com/docs/4.17.15#minBy

Reply:
is 
`_.minBy(data, (element) => element.value )!.value;`
more readable than 
`Math.min(...data.map((d) => d.value));`

Reply:
No dobra, to w sumie gdy w tym momencie jeszcze dojdzie `low` i `high` to możemy zrobić coś takiego:

```js
const minValue = Math.min(...data.map((d) => (d.low ? d.low : d.value)));
const maxValue = Math.max(...data.map((d) => (d.high ? d.high : d.value)));
```